#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_desktop-entry-invalid-category.dpatch by Andrew Lee <andrew@linux.org.tw>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Refine Categories to fit debian menu structure

@DPATCH@
diff -urNad pcmanfm-0.5~/pcmanfm.desktop.in pcmanfm-0.5/pcmanfm.desktop.in
--- pcmanfm-0.5~/pcmanfm.desktop.in	2008-07-16 19:48:37.000000000 +0800
+++ pcmanfm-0.5/pcmanfm.desktop.in	2008-07-17 14:05:52.000000000 +0800
@@ -2,7 +2,7 @@
 Version=1.0
 Encoding=UTF-8
 Name=PCMan File Manager
-Categories=Application;System;Utility;Core;GTK;
+Categories=System;Utility;Core;GTK;
 Name[zh_TW]=檔案總管
 Exec=pcmanfm %F
 Icon=pcmanfm
diff -urNad pcmanfm-0.5~/pcmanfm.desktop.in.rej pcmanfm-0.5/pcmanfm.desktop.in.rej
--- pcmanfm-0.5~/pcmanfm.desktop.in.rej	1970-01-01 08:00:00.000000000 +0800
+++ pcmanfm-0.5/pcmanfm.desktop.in.rej	2008-07-17 14:05:28.000000000 +0800
@@ -0,0 +1,17 @@
+***************
+*** 2,8 ****
+  Version=1.0
+  Encoding=UTF-8
+  Name=PCMan File Manager
+- Categories=Application;System;Utility;Core;GTK;
+  Name[zh_TW]=PCMan 檔案管理程式
+  Name[pl]=Przeglądarka plików z zakładkami
+  Exec=pcmanfm %F
+--- 2,8 ----
+  Version=1.0
+  Encoding=UTF-8
+  Name=PCMan File Manager
++ Categories=System;Utility;Core;GTK;
+  Name[zh_TW]=PCMan 檔案管理程式
+  Name[pl]=Przeglądarka plików z zakładkami
+  Exec=pcmanfm %F
